bleachbit/markovify/utils.py

```diff
@@ -50,14 +50,13 @@ def combine(models, weights=None):
     if isinstance(ret_inst, Chain):
         return Chain.from_json(c)
     if isinstance(ret_inst, Text):
-        if any(m.retain_original for m in models):
```

Comment:
You may want to suggest the same to the upstream project https://github.com/jsvine/markovify